dask_image/imread/__init__.py

```diff
@@ -43,6 +45,12 @@ def imread(fname, nframes=1):
     if (nframes != -1) and not (nframes > 0):
         raise ValueError("`nframes` must be greater than zero.")
 
+    if arraytype == "numpy":
+        arrayfunc = numpy.asanyarray
+    elif arraytype == "cupy":
+        import cupy
+        arrayfunc = cupy.asanyarray
+
     with pims.open(fname) as imgs:
         shape = (len(imgs),) + imgs.frame_shape
         dtype = numpy.dtype(imgs.pixel_type)
```

Comment:
FYI: we don't need to change the line `numpy.dtype(imgs.pixel_type)`, this works fine as is.